include/overset/ExtOverset.h

```diff
@@ -44,15 +44,26 @@ class ExtOverset
   void set_multi_solver_mode(const bool flag)
   { multiSolverMode_ = flag; }
 
+  bool is_external_overset() const
+  { return hasOverset_ && isExtOverset_; }
```

Comment:
Should `hasOverset_` be `true` if `isExtOverset_` is `true` ?

Reply:
Thanks, that is indeed redundant. I had a more complicated logic that I replaced with `isExtOverset_` but overlooked an additional simplification.

Reply:
Fixed in 81c7d132cbc4